client/adventureMap/CAdvMapInt.h

```diff
@@ -110,7 +110,6 @@ class CAdvMapInt : public CIntObject
 
 	std::shared_ptr<MapAudioPlayer> mapAudio;
 
-private:

```

Comment:
IMO old code is better - it was separating two semantically different sections - private members and private methods

Reply:
restored